LayoutTests/fast/parser/strict-mode-in-srcdoc-with-quirks-doctype-expected.txt

```diff
@@ -0,0 +1,14 @@
+This tests that the documents in srcdoc are always in strict mode
+
+On success, you will see a series of "PASS" messages, followed by "TEST COMPLETE".
+
+
+PASS frame1.contentDocument.compatMode is "CSS1Compat"
+PASS frame2.contentDocument.compatMode is "CSS1Compat"
+FAIL frame3.contentDocument.compatMode should be CSS1Compat. Was BackCompat.
```

Comment:
Is this expected? It doesn't seem to be explained in the test or changelog. 

I have verified that this test fully passes in Gecko & Blink. I am guessing that your patch gets us a bit closer to Gecko and Blink but still not fully aligned?

Reply:
oh weird.